src/.editorconfig

```diff
@@ -1,7 +1,7 @@
 # TODO: Uncomment when analyzer line ending problem is fixed
 #root = true
 
-[*.{csproj,xml}]
+[*.{csproj,props,targets,xml}]
```

Comment:
@bording should we add `nuget.config` too?

Reply:
We could, though given that's a file synced here, that seems less important since it can't be edited in repos.

Reply:
repos could have their own props and targets files too though no? E.g. in subdirs

Reply:
I don't understand what you're asking here.

Reply:
Sorry, I got confused between prop and targets files and nuget.config. As you say, there's no point adding this. Unless, of course, it would be to ensure the code style of the file that is maintained in this repo and pushed out to others, but given that this `.editorconfig` is under the `src` directory, it wouldn't have that effect, because `nuget.config` is in the root.

I do wonder if this `.editorconfig` should really be in the root, although then I guess that gets complicated because we keep our `sln` files under `src` (I really wish our `sln` files were in the root).